addons/web/static/src/core/errors/error_dialogs.js

```diff
@@ -19,6 +19,7 @@ export const standardErrorDialogProps = {
     subType: { type: [String, { value: null }], optional: true },
     code: { type: [Number, String, { value: null }], optional: true },
     type: { type: [String, { value: null }], optional: true },
+    serverTime: { type: [String, {value: null}], optional: true },  // TODO: Date type instead ? but we want to make sure to keep the UTC timezone
```

Comment:
I think it's ok with a string. The date is formatted server time, the client simply displays it.

Don't forget to enable the web tooling though, as I see your code isn't properly formatted.